pandas/core/indexes/datetimes.py

```diff
@@ -2245,17 +2245,47 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
     --------
     pandas.period_range : Return a fixed frequency PeriodIndex.
     pandas.interval_range : Return a fixed frequency IntervalIndex.
+    Numpy daterange: `numpy daterange<https://docs.scipy.org/doc/numpy/reference/arrays.datetime.html>`__.
 
     Examples
     --------
-    >>> pd.date_range('2018-10-03', periods=2) # doctest: +NORMALIZE_WHITESPACE
-    DatetimeIndex(['2018-10-03', '2018-10-04'], dtype='datetime64[ns]',
-                   freq='D')
-
-    >>> pd.date_range(start='2018-01-01', end='20180103')
-    ... # doctest: +NORMALIZE_WHITESPACE
-    DatetimeIndex(['2018-01-01', '2018-01-02', '2018-01-03'],
-                    dtype='datetime64[ns]', freq='D')
+
+    Given the `start` and `end` mandatory parameters,
+    with default ``freq='D'`` (Days)
+
+    >>> pd.date_range(start='1/1/2018', end='1/10/2018')
+    DatetimeIndex(['2018-01-01', '2018-01-02', '2018-01-03', '2018-01-04',
+                   '2018-01-05', '2018-01-06', '2018-01-07', '2018-01-08',
+                   '2018-01-09', '2018-01-10'],
+                  dtype='datetime64[ns]', freq='D')
+
+    Given the ``start`` and ``periods`` mandatory parameters,
+    periods would be the limit of the DatetimeIndex, with default `freq='D'`
```

Comment:
I'm not sure what you mean by "would be the limit."

I would remove everything after "parameters" on the line above. You've already explained the default freq.

Reply:
update this documentation.